.markdownlintignore

```diff
@@ -1,2 +1,5 @@
 # Ignore submodule
 bellman-cuda
+
+# Ignore Github configuration markdowns
+.github
```

Comment:
I think it would be better to ignore the particular line rather than ignore a whole directory. An example could be found in the [configuration section of the markdownlint project](https://github.com/DavidAnson/markdownlint/blob/main/README.md#configuration).

Reply:
I agree with your approach. The problem with this approach, however, is that any comments in `.github` file are treated as a raw Github template. If we were to add such comment that ignores linting on `.github/pull_request_template.md`, Github PR will be opened with that comment in the template. 
Instead of having users manually delete them, or having those lines be included in PRs, I decided to mark the entire `.github` directory as an exception.